### PR TITLE
Add actual date and time to 'timeago' dates (#5085)

### LINKED ISSUE #5085: Add actual date and time to 'timeago' dates
As per feedback here:

Right now, we display a humanized version of the date / time.  

As per feedback here: https://github.com/pypa/warehouse/pull/4402#issuecomment-410383106

> sometimes, I forget when I have released my package since I have 40+ packages. So I resorted to look at pypi for more info. However, github releases and pypi at moment both love moment.js and also give some time ago. Still I could not find the exact date when a version is released.

For this, I think we should add a tooltip that shows *on hover*, e.g.

screenshot from 2018-11-18 15-14-36

The datetime in the tooltip should follow an international format, e.g `YYYY-MM-DD hh:mm`

this would also solve https://github.com/pypa/warehouse/issues/3010 by providing both a "human friendly" format and technical format.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

**Screenshot Required**: Please include a screenshot of your update. This helps our team give you feedback faster.

### PR DESCRIPTION
**Pull request by darkHarry**
Fixes #5085.

As per @dstufft every date time is stored in UTC so I added "UTC" in the end. Let me know if that's not what I was supposed to do.
* Added a tooltip for date time to 'timeago' dates with timezone

### CODE REVIEW

**Comment by di** [comment]
I think the right thing to do here would be to localize the datetime prior to using it, and then add `%Z` to the formatting string:

```python
>>> dt
datetime.datetime(2018, 12, 26, 13, 36, 5, 789013)
>>> import pytz
>>> pytz.utc.localize(dt)
datetime.datetime(2018, 12, 26, 13, 36, 5, 789013, tzinfo=<UTC>)
>>> _.strftime('%Y-%m-%d %H:%M %Z')
'2018-12-26 13:36 UTC'
```

**Comment by darkHarry** [comment]
Hi @di! Thanks for the review. I know printing "UTC" was plain stupid :P 
I actually ran into a problem regarding the all_releases query object in /warehouse/packaging/views.py which passes the all_releases list to the template, as it can't be modified so I converted it to a dict but then 1 test failed and moreover, we can't import pytz in the template. Any help appreciated

**Comment by di** [comment]
I'd recommend adding it [as a filter](https://github.com/pypa/warehouse/blob/1fbb4ac752e68b5840b9e09b68e44a165569bfa6/warehouse/filters.py) instead, so we could use it in the Jinja template whenever we need to:

```
{{ hversion.created.strftime('%Y-%m-%d %H:%M') | localize_datetime }}

**Comment by darkHarry** [comment]
Thanks, @di!
I've added a filter to localize datetime and its corresponding unit test. 
Also, I found another filter to format datetime so, i removed the ``strftime()`` and used that instead.

File: warehouse/filters.py [hunk 1]
```diff
@@ -156,3 +157,7 @@ def parse_version(version_str):
 
 def includeme(config):
     config.add_request_method(_camo_url, name="camo_url")
+
+
+def localize_datetime(timestamp):
+    return pytz.utc.localize(timestamp)
```
[the remark this answers is not in the document]
Comment (darkHarry) [reply warehouse/filters.py]: sure

**Comment by di** [comment]
Thanks @darkHarry!

**Comment by darkHarry** [comment]
It was Awesome. Thanks a lot, @di.

**Pull request merged**